src/controllers/hotkeys/HotkeyModel.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+#include "common/SignalVectorModel.hpp"
+#include "controllers/hotkeys/Hotkey.hpp"
+#include "util/QStringHash.hpp"
```

Comment:
```suggestion
```

Reply:
Won't build with this suggestion, QStringHash seems to be required for the constructor.